dbms/programs/client/readpassphrase/readpassphrase.c

```diff
@@ -121,7 +121,6 @@ readpassphrase(const char *prompt, char *buf, size_t bufsiz, int flags)
 	(void)sigaction(SIGPIPE, &sa, &savepipe);
 	(void)sigaction(SIGQUIT, &sa, &savequit);
 	(void)sigaction(SIGTERM, &sa, &saveterm);
-	(void)sigaction(SIGTSTP, &sa, &savetstp);
```

Comment:
This can remain in `readpassphrase` function, because it is used only in client in interactive mode where other (server) signals are not processed.